packages/amplication-server/src/core/build/build.controller.ts

```diff
@@ -27,9 +26,9 @@ export class BuildController {
     EnvironmentVariables.instance.get(CHECK_USER_ACCESS_TOPIC, true)
   )
   async checkUserAccess(
-    @Payload() message: KafkaMessage
+    @Payload() message: CanUserAccessArgs
```

Comment:
is `plainToInstance` still needed? 
Nestjs will attempt to parse the message to JSON and the right type here